src/View/Helper/FormHelper.php

```diff
@@ -120,6 +120,10 @@ public function input($fieldName, array $options = [])
             case 'select':
             case 'multiselect':
             case 'textarea':
+                if (isset($options['multiple']) && $options['multiple'] === 'checkbox') {
```

Comment:
Maybe this should be limited to the `select` case? By default I think cake will assign the `select` to `$options['type']` given there is an array for `$options['options']`.